examples/servicenow_examples.md

```diff
@@ -11,49 +11,50 @@ added later.
 ```python
```

Comment:
In the end I have rewritten this file a bit copy-pasting parts of bugzilla_examples.md (btw there was 'RT ticket' on line 30) and updated servicenow.py, so set_ticket_id works fine.